zerver/lib/upload/s3.py

```diff
@@ -402,12 +402,13 @@ def upload_single_emoji_image(
     @override
     def get_export_tarball_url(self, realm: Realm, export_path: str) -> str:
         client = self.avatar_bucket.meta.client
+        if export_path.startswith("/"):
+            export_path = export_path.removeprefix("/")
```

Comment:
The `startswith` check is unnecessary here.

Reply:
Yeah, `removeprefix` removes the prefix if and only if it is present.

Reply:
Fixed it.